addons/crm/models/crm_lead.py

```diff
@@ -316,12 +316,12 @@ def _compute_stage_id(self):
     @api.depends('user_id')
     def _compute_date_open(self):
         for lead in self:
-            lead.date_open = fields.Datetime.now() if lead.user_id else False
+            lead.date_open = self.env.cr.now() if lead.user_id else False
 
     @api.depends('stage_id')
     def _compute_date_last_stage_update(self):
         for lead in self:
-            lead.date_last_stage_update = fields.Datetime.now()
+            lead.date_last_stage_update = self.env.cr.now()
```

Comment:
Side note: we could also update date_closed computation in crm_lead file, just to be coherent (not having any impact on tests but while we are cleaning it, maybe we could cleanup all datetime fields).